SinusSynchronous/UI/CharaDataHubUi.GposeTogether.cs

```diff
@@ -84,13 +86,13 @@ private void DrawGposeTogether()
         {
             if (_uiSharedService.IconTextButton(FontAwesomeIcon.ArrowUp, "Send Updated Character Data"))
             {
-                _ = _charaDataGposeTogetherManager.PushCharacterDownloadDto();
+                _ = _charaDataGposeTogetherManager.PushCharacterDownloadDto(_selectedServerIndex);
             }
             UiSharedService.AttachToolTip("This will send your current appearance, pose and world data to all users in the lobby.");
             if (!_uiSharedService.IsInGpose)
             {
                 ImGuiHelpers.ScaledDummy(5);
-                UiSharedService.DrawGroupedCenteredColorText("Assigning users to characters is only available in GPose.", ImGuiColors.DalamudYellow, 300);
+                UiSharedService.DrawGroupedCenteredColorText("Assigning users to characters is only avail   able in GPose.", ImGuiColors.DalamudYellow, 300);
```

Comment:
```suggestion
                UiSharedService.DrawGroupedCenteredColorText("Assigning users to characters is only available in GPose.", ImGuiColors.DalamudYellow, 300);
```
I don't think this was supposed to happen, no?